### ISSUE TITLE
Houdini grid overlapping render output

### ISSUE THREAD

**Issue by dedoardo** [tangent-opensource/hdcycles#111]
The world grid overlay (with transparency) in Houdini overlaps the rendering output. I would guess the depth buffer needs to accessible to Houdini to correctly compose the two images.

Would be nice to fix, though not an high priority.

![Capture](https://user-images.githubusercontent.com/2072636/112228490-c4d6c000-8c07-11eb-8830-8522185a6e1d.PNG)

**Comment by boberfly** [comment]
Yeah the depth AOV needs added, and I think this will help camera movement also as it's not working right either. I'd like to move the priority up but we can get by with GL mode as a workaround...

**Comment by dedoardo** [comment]
After PR #162, this has become only a Linux specific issue.
